apps/server/src/env.mjs

```
@@ -33,15 +45,30 @@ const server = z.object({
+  ALERT_SMS_DISABLED: coerceBooleanWithDefault(true),
+  ALERT_WHATSAPP_DISABLED: coerceBooleanWithDefault(true),
+  
   // API caching configuration. Enabled by default in production, disabled in development.
   PUBLIC_API_CACHING: z.union([z.literal("true"), z.literal("false")]).optional()
     .transform((val) => {
       if (val === undefined) return process.env.NODE_ENV === "production";
       return val === "true";
 
+
+
+  PUBLIC_API_CACHING: z
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)